### PR TITLE
Add upgrade script for v8 to update MUI packages and run MUI codemods

### PR DESCRIPTION
**Pull request by jamesricky**
## Related tasks and documents

Task: https://vivid-planet.atlassian.net/browse/COM-1123
MUI Update in Comet: https://github.com/vivid-planet/comet/pull/2592

### CODE REVIEW

File: src/util/update-dependency-version.util.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { readFile, writeFile } from "fs/promises";
+
+import { formatCode } from "./format-code.util";
+
+export async function updateDependencyVersion(packageJsonPath: string, packageName: string, newVersion: string) {
+    const fileContent = (await readFile(packageJsonPath)).toString();
+    const packageJson = JSON.parse(fileContent);
```
Comment (johnnyomair) [inline src/util/update-dependency-version.util.ts:7]: Should we check if the file exists? Otherwise this might fail if there's no Admin

Comment (jamesricky) [reply src/util/update-dependency-version.util.ts:7]: Good point, done in 00bfbf2eea89b2def42ceacb0f892e9de0f60243

**Review by johnnyomair — APPROVED** [review]

**Comment by jamesricky** [comment]
@johnnyomair can we merge this? 

Or should we wait for the actual release of v8 as the script would currently fail as the version does not exist? 
Should we create a `feature/v8` or `next` branch for this?

**Comment by johnnyomair** [comment]
> vivid-planet/comet#2592

I wanted to wait until https://github.com/vivid-planet/comet/pull/2592 has been merged.

**Pull request merged**